src/shared/services/usersService.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import Item from '../models/item';
+
+export class AddUserError extends Error {
+    userToAdd: Item;
+    constructor(message: string, userToAdd: Item) {
+        super(message);
+        this.userToAdd = userToAdd;
+    }
+}
+
+export const getUsers = () => new Promise((resolve, reject) => axios
+    .get('/api/users')
+    .then(response => resolve(response.data))
+    .catch(reason => reject(reason))
+);
+
+export const getTeamMembers = (teamId: string) => new Promise((resolve, reject) => axios
```

Comment:
I think this should be in the teams service

Reply:
Moved to the teams service